src/utils/userData.ts

```diff
@@ -1,18 +1,17 @@
-import { User } from "../types/User";
-import { Cookies } from "react-cookie";
+import { User, UserResult } from "../types/User";
+import { HttpStatusCode } from "../types/StatusCodeEnums";
+
+export const getUserData = async (): Promise<UserResult> => {
+  const userResult: UserResult = {
+    user: undefined,
+    statusCode: HttpStatusCode.InternalServerError,
+  };
 
-export const getUserData = async (): Promise<User | undefined> => {
   try {
     const response = await fetch(`/api/v3/user/me`);
```

Comment:
Also very danger. In 2 sprints i expect that this endpoint is removed. A new one is under developing api/v3/me. But by creating the task, nobody know that it is used in tldraw client.